docs/pages/choose-an-edition/migrate-to-cloud.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: Migrating from Teleport Enterprise (Self-Hosted) to Teleport Enterprise (Cloud-Hosted)
+description: How to migrate from Teleport Enterprise (self-hosted) to Teleport Enterprise (cloud-hosted).
+h1: Migrate to Cloud-Hosted Teleport Enterprise 
+---
+
+Migrating from a Teleport Enterprise (self-hosted) deployment to Teleport
+Enterprise (cloud-hosted) offers scalability, reliability, and ease of
+management. By following the steps outlined in this guide, you can transition
+your Teleport Enterprise (self-hosted) deployment to Teleport Enterprise
+(cloud-hosted) successfully while ensuring minimal disruption to your
+operations.

```

Comment:
I'm probably yelling into the void and the decision has already been made above my pay grade, but I really don't like this whole "same product name with different qualifier in brackets" style. It's just clunky and awkward to read.